src/App.jsx

```diff
@@ -1,104 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsById(postIndex) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postIndex) || null
+  );
+}
```

Comment:
The functions `getUserById` and `getCommentsById` can be simplified because find returns `undefined` if no user is found and filter returns an `empty array`.

```suggestion
function getUserById(userId) {
  return usersFromServer.find(user => user.id === userId);
}

function getCommentsById(postIndex) {
  return (
    commentsFromServer.filter(comment => comment.postId === postIndex);
}
```